app/javascript/src/components/Dashboard/Notes/Pane/Form.jsx

```diff
@@ -3,24 +3,30 @@ import React from "react";
 import { Formik, Form as FormikForm } from "formik";
 import { Button, Pane, Toastr } from "neetoui";
 import { Input, Textarea, Select } from "neetoui/formik";
+import * as R from "ramda";
```

Comment:
If you are using Ramda then please don't import as `*`. But rather only import the necessary functions from it.

Also if you are using Ramda, then refer: https://academy.bigbinary.com/learn-react/react-best-practices/learn-ramda-js